Dockerfile

```diff
@@ -32,12 +26,9 @@ LABEL org.opencontainers.image.source https://github.com/kubewarden/policy-serve
 
 USER root
 
-COPY --from=sle /etc/passwd /etc/passwd
-COPY --from=sle /*.rpm /
+COPY --from=builder /etc/passwd /etc/passwd
```

Comment:
I wonder if this is adding more users than needed, now that it happens from the openSUSE image.

Reply:
To be honest, adding a user to the container can be done with a simple `echo` command:

```shell
# By default we will run as this user...
RUN echo "policy-server:x:65533:65533::/tmp:/sbin/nologin" >> /etc/passwd
# Add the default GID to /etc/group for completeness.
RUN echo "policy-server:x:65533:policy-server" >> /etc/group
```

Taken from [here](https://github.com/kubernetes/git-sync/blob/dc324f42839a6b2fb28f0305bc48c7b4dbb9fe0a/Dockerfile.in#L68-L75)

Reply:
Done, thank you!